migrations/migrations.sql

```diff
@@ -0,0 +1,34 @@
+/*Just copying over the data from our struct models*/
+
+CREATE TABLE IF NOT EXISTS courses (
+    id INT PRIMARY KEY AUTO_INCREMENT,
+    department VARCHAR(255) NOT NULL,
+    course_number INT NOT NULL,
+    subsection INT NOT NULL, 
+    title VARCHAR(255) NOT NULL
+);
+
+
+CREATE TABLE IF NOT EXISTS reviews (
+    id INT PRIMARY KEY AUTO_INCREMENT,
+    rating INT NOT NULL,
+    
+	/*Does it make sense to keep these as optional inputs?*/
```

Comment:
difficulty and interest are the two things we want mandatory for a review, so NOT NULL is okay with these